.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -9,7 +9,8 @@ body:
       description: What Apache Iceberg version are you using?
       multiple: false
       options:
-        - "1.2.1 (latest release)"
+        - "1.3.0 (latest release)"
+        - "1.2.1"
         - "1.2.0"
         - "1.1.0"
         - "1.0.0"
```

Comment:
nit: can we remove the older versions? (maybe before 1.0.0) 
Just to discourage users from reporting issues from older versions.

Reply:
I would leave them in for now, since there are lots of people still running older versions

Reply:
yeah, but they have to mostly try it on the newer version to check whether it is already fixed or not. Since lot of code is changed. 

We can maybe remove older versions during next release update.